apps/src/code-studio/progressRedux.js

```diff
@@ -212,9 +212,14 @@ export const setIsSummaryView = isSummaryView => ({ type: SET_IS_SUMMARY_VIEW, i
 // Do we have one or more lockable stages
 export const hasLockableStages = state => state.stages.some(stage => stage.lockable);
 
-export const lessonNames = state => state.stages.map(stage => stage.name);
 export const hasGroups = state => Object.keys(categorizedLessons(state)).length > 1;
 
+const lessonFromStage = stage => ({
+  name: stage.name,
+  id: stage.id
+});
+export const lessons = state => state.stages.map(lessonFromStage);
```

Comment:
Instead of returning just names, return a set of "lesson" objects, that has both name and id (and will likely need at least one more field once I do lockable stages)